src/components/Navigation/Navigation.scss

```
@@ -26,15 +27,28 @@ $nav-max-width: rem(360px);
     border-right: var(--p-override-none, border());
     @include safe-area-for(max-width, layout-width(nav), left);
   }
+
+  &.Navigation-globalTheming {
+    max-width: $unstable-nav-max-width;
+    border-right: 1px solid var(--p-border-disabled);
```

```suggestion
    border-right: 1px solid var(--p-border-subdued);
```

Disabled should be reserved only for disabled elements.